### DISCUSSION TITLE
[Feature] Automatic album share via Facial Recognition

### DISCUSSION THREAD

**Discussion by huntervcx** [immich-app/immich#12913] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

The ability that an album is automatically shared to immich users that appears on

Different of the partner sharing
For example, i host my family pictures. I do albums for holidays, that i share with them each time. It would be great that all my albums containing at least X pictures of them, are automatically shared to their immich accounts

Or maybe at least a "quick share" to all users with their face recognized in the album could pop 

### Platform

- [X] Server
- [X] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
#1673
